### PR TITLE
ES modules

### PR DESCRIPTION
**Pull request by dpup**
#### What's this PR do?

Updates the library to support ES Modules.

#### Definition of Done:

- [ ] Code is easy to understand and conforms with Prettier & eslint configs
- [ ] Incomplete code is marked with TODOs
- [ ] Code is suitably instrumented with logging and metrics
- [ ] Documentation has been updated as appropriate
- [ ] Manifest has been updated and version incremented correctly
- [ ] [OWASP Top 10](https://www.owasp.org/index.php/Top_10-2017_Top_10) have been considered

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,5 +18,10 @@
     "semi": true,
     "singleQuote": true,
     "quoteProps": "consistent"
-  }
+  },
+  "directories": {
+    "lib": "lib"
+  },
+  "main": "lib/range.js",
+  "type": "module"
```
Comment (ghost) [inline package.json:26]: @dpup These lines are necessary to get https://github.com/range-labs/range-export working correctly. And judging from the version change above to 1.4.0, I think this was published to npm for the past year.

File: lib/range.js [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
 
-const https = require('https');
-const querystring = require('querystring');
+import https from 'https';
+import querystring from 'querystring';
+
+// TODO: Figure out proper way to import json in ES modules.
+import { createRequire } from 'module';
+const require = createRequire(import.meta.url);
+const PACKAGE_VERSION = require('../package.json').version;
```
Comment (ghost) [inline lib/range.js:9]: I added this, which works for now.

**Review by ghost — APPROVED** [review]
Any problem with me publishing this?

**Comment by dpup** [comment]
Go for it. Thanks.

**Pull request merged**